web/src/components/apps/AppVersionHistory.tsx

```diff
@@ -81,69 +81,69 @@ type Props = {
 } & withRouterType;
 
 type State = {
-  logsLoading: boolean;
```

Comment:
going to make a lot of changes to this and i want the diffs to be clear. so sorting this now and not part of the bigger changes.